l10n_ro_hr_payroll/models/hr_contract.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#     Author:  Adrian Vasile <adrian.vasile@gmail.com>
+#    Copyright (C) 2014 Adrian Vasile
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, fields, api, _
+
+class hr_advatages(models.Model):
+    _name = 'hr.advantages'
+    _description = 'Advantages'
+
+    company = fields.Many2one('res.company', _('Company'), required = True)
```

Comment:
Adi, de ce folosesti _( in loc de string='...', la ce te ajuta...ca nu am chef acuma sa caut...